recipes-debian/glibc/locales_debian.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "locales utils -- locale-gen, update-locale, validlocale"
+LICENSE = "LGPL-2.1"
+
+LIC_FILES_CHKSUM = "file://LICENSES;md5=cfc0ed77a9f62fa62eded042ebe31d72 \
+      file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+      file://posix/rxspencer/COPYRIGHT;md5=dc5485bb394a13b2332ec1c785f5d83a \
+      file://COPYING.LIB;md5=4fbd65380cdd255951079008b364516c"
+
+inherit debian-package
+require recipes-debian/sources/glibc.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/glibc-${PV}"
+
+RDEPENDS_${PN} = "perl \
+                  perl-module-getopt-long \
+                  perl-module-posix \
+                  perl-module-carp \
+"
+
+do_compile[noexec] = "1"
+
+do_install() {
+    install -d ${D}/${sbindir}
+    install -m 0755 ${S}/debian/local/usr_sbin/* ${D}/${sbindir}
+
+    install -d ${D}/${datadir}/locale/
+    install -m 0644 ${S}/intl/locale.alias ${D}/${datadir}/locale/
+}
+
+do_poststash_install_cleanup_append() {
```

Comment:
I think this doesn't work because do_poststash_install_cleanup() existed in meta/recipes-core/glibc/glibc-package.inc. And this recipe doesn't include the glibc-package.inc.

In addition, do_poststash_install_cleanup() was removed on the latest warrior branch yesterday, so we can't rely on it anyway.
http://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?h=warrior&id=b174d936e91902c3e2ab800856bb7bc492d096a0